Add information about building (#277)
TYPO3-Documentation/TYPO3CMS-Guide-ContributionWorkflow: Documentation/Build/Index.rst

. include:: /Includes.rst.txt

.. highlight:: bash

.. index::
   single: Code Contribution Workflow; Build
   single: Build
   single: SCSS
   single: TypeScript
.. _build:

=====

> Build

Reviewer: If we focus on assets then maybe **Building assets** would be a more descriptive title.

Author: I pushed another commit to fix make suggested changes.

=====

> This page explains building JavaScript and CSS files.

Reviewer: ```suggestion
This page explains building assets like JavaScript and CSS files.
```

For the most part, the TypeScript, JavaScript and SCSS sources are contained in

> :file:`Build/Sources`. These are converted into target files, usually located in:

Reviewer: ```suggestion
:file:`Build/Sources`. These are compiled to target files, usually located in:
```
